readme.md

```diff
@@ -2,15 +2,16 @@
 
 1. create database **'edge-node-auth-service'**
 2. cp .env.example .env
-3. generate random strings for following .env variables: **JWT_SECRET** and **SECRET** (you can use openssl rand -hex 64 for example)
+3. generate random strings for following .env variables: **JWT_SECRET** and **SECRET** (you can use openssl rand -hex 64 for example) + Add MySQL password to **DB_PASSWORD** variable if necessary
 4. **npm install**
 5. npx sequelize-cli **db:migrate**
 6. npx sequelize-cli **db:seed:all** (this will generate demo user)
-7. define Edge node config parameters by executing **provided MYSQL query ([UserConfig.sql](UserConfig.sql))**, updated with real values
+7. Update **this MYSQL query ([UserConfig.sql](UserConfig.sql))** in your cloned repository with real values of:
```

Comment:
are these the only values that we need to update, I kinda remember being more of them but maybe I'm mistaken

Reply:
I think they are but if you find something please let me know and I will include it